File: chapter_one/UN_CC_ASSIGNMENTS/INPROG01/EOMUL.c
// https://www.codechef.com/INPRG01/problems/EOMUL

#include <stdio.h>

int main()
{
  int j;
  scanf("%d", &j);

  if (j % 3 != 0){
    printf("%d", -1);
  }

  if (j % 3 == 0 && j % 2 != 0){
    printf("%d", 1);
  }

  if (j % 3 == 0 && j % 2 == 0){
    printf("%d", 0);
  }

}

File: chapter_one/CODECHEF/UNACADEMY_ASSIGNMENTS/INPROG01/EOMUL.py
# https://www.codechef.com/INPRG01/problems/EOMUL

j = int(input())

if (j % 3 != 0):
    print(-1)

if (j % 3 == 0) and (j % 2 != 0):
    print(1)

if (j % 3 == 0) and (j % 2 == 0):
    print(0)

# (c) 2021, Vikramaditya V. Vajire
# https://github.com/vikramaditya
# This source code is licensed under the MIT License
